services/connector/commands/sign.go

```diff
@@ -33,31 +39,40 @@ func (r *RPCRequest) getPersonalSignParams() (*PersonalSignParams, error) {
 		return nil, ErrInvalidParamsStructure
 	}
 
+	challengeIndex := 0
```

Comment:
I am not sure when exactly we moved away from the key/value map to the array, cause the previous approach, which is still in the develop branch https://github.com/status-im/status-go/blob/develop/services/connector/commands/personal_sign.go#L27-L53 is order resistant and more unique.

@alexjba do you think that we should move back to it?